ts/components/screens/DarkLayout.tsx

```diff
@@ -30,8 +30,8 @@ type Props = Readonly<{
   allowGoBack?: boolean;
   headerBody?: React.ReactNode;
   title?: string;
```

Comment:
on the same line as one of my previous comments, I believe refactoring to FC syntax would make the code style more consistent with the rest of the app; maybe another tech debt task?